generatedTypes/index.d.ts

```diff
@@ -67,7 +68,6 @@ export {default as Carousel, CarouselProps} from './components/carousel';
 export {
   ActionSheet,
   ConnectionStatusBar,
-  Drawer,
```

Comment:
Make sure to update the exports also in `src/index.ts` file

Reply:
Done